src/Blorc.OpenIdConnect/Models/Profile.cs

```diff
@@ -1,10 +1,45 @@
 ﻿namespace Blorc.OpenIdConnect
 {
+    using System.Collections.Generic;
     using System.Security.Claims;
+    using System.Text.Json;
     using System.Text.Json.Serialization;
 
     public class Profile
     {
+        [ClaimType("aud")]
+        [JsonPropertyName("aud")]
+        public JsonElement Aud
+        {
+            get;
+            set;
+        } = JsonDocument.Parse("[]").RootElement;
```

Comment:
Do we need to recreate / parse every time or can we use a static readonly cache object as empty array?

Reply:
I think it'll be better to create every time for reference isolation. However, if we can use "required" for breaking changes, it should be no longer needed. Let me have a try.